src/SSHDebugPS/ConnectionManager.cs

```diff
@@ -35,6 +34,16 @@ public static DockerConnection GetDockerConnection(string name)
             // Assume format is <server>/<hostname>::<container> where if <server> is specified, it is for SSH
             string[] connectionStrings = name.Split('/');
 
+            // Format is wrong, ask the user to select from the dialog
+            if (connectionStrings.Length > 2 || connectionStrings.Length < 1)
+            {
+                string connectionString;
+                if (ShowContainerPickerWindow(IntPtr.Zero, out connectionString))
+                {
+                    return GetDockerConnection(connectionString);
+                }
```

Comment:
>} [](start = 16, length = 1)

if this does not return true, do you really want to fall through to the rest of this function?

Reply:
Oh, I see, maybe you do and are trying to fall through to the error block? Might be worth a comment if so.

---
In reply to: [301297533](https://github.com/microsoft/MIEngine/pull/891#discussion_r301297533) [](ancestors = 301297533)